tests/baselines/reference/tscWatch/programUpdates/when-changing-`allowImportingTsExtensions`-of-config-file.js

```diff
@@ -47,12 +47,7 @@ DirectoryWatcher:: Added:: WatchInfo: /user/username/projects/myproject/node_mod
 Elapsed:: *ms DirectoryWatcher:: Added:: WatchInfo: /user/username/projects/myproject/node_modules/@types 1 undefined Type roots
 DirectoryWatcher:: Added:: WatchInfo: /user/username/projects/node_modules/@types 1 undefined Type roots
 Elapsed:: *ms DirectoryWatcher:: Added:: WatchInfo: /user/username/projects/node_modules/@types 1 undefined Type roots
-[96mb.ts[0m:[93m1[0m:[93m8[0m - [91merror[0m[90m TS5097: [0mAn import path can only end with a '.ts' extension when 'allowImportingTsExtensions' is enabled.
```

Comment:
This test case needs to change to import * as a from "./a.ts" so the errors are shown

Reply:
Why? Isn't this effectively for #58791 about side effect imports, which are not resolved today without #50394/#58941?

Reply:
This was testing "allowTsExtension" has effect on change and i would prefer if there are errors before and after so that its clear

Reply:
Aha, yes, gotcha.

Reply:
In that case I would also actually duplicate the test and keep the side-effect import, since this did seem to show a bug.

Reply:
I added the duplicated test case that verifies the error goes away with a change from `allowImportingTsExtensions: false` to `allowImportingTsExtensions: true`